lib/structures/Message.js

```diff
@@ -244,6 +249,10 @@ class Message extends Base {
             case MessageTypes.AUTO_MODERATION_ACTION: {
                 break;
             }
+            case MessageTypes.ROLE_SUBSCRIPTION_PURCHASE: {
+                // I have no clue what the message content is for this, any help would be appreciated
```

Comment:
@bsian03 This PR isn't affected by your changes in #1485 afaik. I just need to know the message content for the role subscription message that Discord sends, cuz I assume there is a message that gets sent for it?

Reply:
Depends on what the role subscription purchase type is. These are the possible values:
```
{username} joined **{tierName}** as a subscriber of {guildName}!
{username} joined **{tierName}** and has been a subscriber of {guildName} for {months, plural, =1 {# month} other {# months}}!
{username} renewed **{tierName}** in their {guildName} subscription!
{username} renewed **{tierName}** and has been a subscriber of {guildName} for {months, plural, =1 {# month} other {# months}}!
```
Honestly I think it'd be easier if it was left blank, but it's up to you if you want to include them

Reply:
I opted for including all possible contents for consistency 👍🏻